fedlearner-sgx-dev.dockerfile

```diff
@@ -60,8 +62,8 @@ RUN apt-get install -y libcurl4-openssl-dev libprotobuf-c-dev python3-protobuf p
 RUN apt-get install -y libgmp-dev libmpfr-dev libmpc-dev libisl-dev nasm protobuf-compiler
 
 RUN ln -s /usr/bin/python3 /usr/bin/python \
-    && pip3 install --upgrade pip \
-    && pip3 install toml meson pyelftools
+    && pip3 install --no-compile --upgrade pip -i https://mirrors.aliyun.com/pypi/simple/ \
```

Comment:
Is it necessary to use mirror from aliyun?

Reply:
If we do not include Aliyun, the compilation and testing on our side are too slow. Therefore, we have added it to anything related to pip, which also ensures the consistency of dependency sources.